platform/ios/src/MGLMapView.mm

```diff
@@ -4656,6 +4749,7 @@ - (MGLAnnotationImage *)imageOfAnnotationWithTag:(MGLAnnotationTag)annotationTag
 
 - (void)deselectAnnotation:(id <MGLAnnotation>)annotation animated:(BOOL)animated
 {
+    MGLLogDebug(@"Deselecting annotation: %@ animated: %@", annotation, NSStringFromBOOL(animated));
```

Comment:
Some of these debug statements are being placed above early returns, so the wording of the debug message can be misleading when looking at logs.